queries/special.py

```diff
@@ -169,6 +169,10 @@ def _play_jazz(qs, q):
     "answer": "Ekkert mál, kæri notandi."
 }
 
+_MIDEIND_DESC = {
+    "Flotta teymið hjá Miðeind skapaði mig."
```

Comment:
Why the curly braces? This parses as a set object.

Reply:
Whoops, silly me.